css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,135 @@
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-3;
+	grid-template-columns: max-content auto max-content;
+	outline-color: $text; // Provided explicitly because outline-color isn't passing through tab-container's shadow DOM
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	.segmented-control-previous,
+	.segmented-control-next {
+		display: flex;
+		align-items: center;
+		padding: $sc-padding-sm;
+		border: $sc-border;
+		border-radius: $sc-border-radius-rounded;
+		background-color: $sc-active-background-color;
+		color: $sc-active-color;
+		font-weight: $weight-semibold;
+		cursor: pointer;
+
+		&:hover {
+			border-color: $sc-active-hover-border-color;
+			background-color: $sc-active-hover-background-color;
+
+			@include forced-colors {
+				border-color: Highlight;
+				background-color: Highlight;
+				color: HighlightText;
+				forced-color-adjust: preserve-parent-color;
+			}
+		}
+	}
+
+	.segmented-control-list {
+		grid-column: 2;
+		display: flex;
+		align-items: center;
+		justify-content: flex-start;
+		border: $sc-border;
+		border-radius: $sc-border-radius-rounded;
+		overflow-x: hidden;
+	}
+
+	.segmented-control {
+		display: flex;
+		position: relative;
+		align-items: center;
+		justify-content: center;
+		padding: $sc-padding-sm;
+		border: none;
+		border-radius: $sc-border-radius-rounded;
+		background-color: transparent;
+		color: $sc-font-color;
+		font-weight: $weight-semibold;
+		cursor: pointer;
+		appearance: none;
+		vertical-align: top;
+
+		@include focus-visible() {
+			outline-color: $text-invert;
+			outline-offset: -$focus-outline-offset;
+		}
+
+		&:hover {
+			background-color: $sc-hover-background-color;
+			z-index: $zindex-hover;
+
+			@include forced-colors {
+				background-color: Highlight;
+				color: HighlightText;
+				forced-color-adjust: preserve-parent-color;
+			}
+		}
+
+		&[aria-selected='true'] {
+			background-color: $sc-active-background-color;
+			color: $sc-active-color;
+			z-index: $zindex-active;
+
+			&:hover {
+				background-color: $sc-active-hover-background-color;
+				z-index: $zindex-hover;
+			}
+
+			@include forced-colors {
+				background-color: Highlight;
+				color: HighlightText;
+				forced-color-adjust: preserve-parent-color;
+			}
+		}
+	}
+
+	&.segmented-controls-lg {
+		font-size: $font-size-5;
+	}
+
+	@include tablet {
+		.segmented-control,
+		.segmented-previous,
+		.segmented-next {
+			padding: $sc-padding;
+		}
+	}
+	@include desktop {
+		.segmented-control,
+		.segmented-previous,
+		.segmented-next {
+			padding: $sc-padding-lg;
+		}
+	}
+}
+
+// After Tabpanels footer content
+
+.segmented-controls-footer {
```

Comment:
It should be placed within the `segmented-controls` as well. I do understand why you placed it that way, but what about changing markup a little intead - will moving `segmented-controls` to `tab-container` work?

Reply:
There are some underlying reasons for the markup structure the way it is in this case. The `segmented-controls` ends with the top pill area.  The tab panels are also not included in the class.  So the footer is a sibling, as are the panels, to the segmented-controls. 

Maybe an outline-color atomic can mitigate this one need?

Reply:
@wibjorn  I would like to leave this CSS in for now as I am doing logic follow up on the footer options.

Reply:
Let's remove it. We don't need styles for things that we aren't yet supporting.